### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by SimaLoktseva**

### CODE REVIEW

File: Names/Models/User.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Names.Models
+{
+   public class User
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
```
Comment (ohthatbaguk) [inline Names/Models/User.cs:6]: blank line between properties

File: Names/Users.cs [hunk 2]
```diff
@@ -1,40 +1,34 @@
...
+        }
+        public static void Main(string[] args)
+        {
+            string jsonString = File.ReadAllText(Names.fileName);
+            var restoredUser = JsonSerializer.Deserialize<User[]>(jsonString);
+            Console.WriteLine("Before sort:");
+            foreach (var user in restoredUser)
             {
-                System.Console.WriteLine($"{ u.FirstName} {u.LastName}");
+                Console.WriteLine($"{ user.FirstName} {user.LastName}");
```
Comment (ohthatbaguk) [inline Names/Users.cs:22]: Console.WriteLine($"{ user.FirstName} {user.LastName}"); - it's bad practice
  you can create a method that outputs the full name

File: Names/Users.cs [hunk 3]
```diff
@@ -1,40 +1,34 @@
...
             }
-
-            var result = restoredUser.OrderBy(u => u.FirstName).ThenBy(u => u.LastName);
-
-            System.Console.WriteLine("---------------------------------------");
-            System.Console.WriteLine("After sort:");
-
-            foreach (User u in result)
+            var result = restoredUser.OrderBy(user => user.FirstName).ThenBy(user => user.LastName);
```
Comment (ohthatbaguk) [inline Names/Users.cs:24]: var sortedUsers = ...

File: Names/Users.cs [hunk 4]
```diff
@@ -1,40 +1,34 @@
...
+            Console.WriteLine("--------------------------");
+            Console.WriteLine("After sort:");
+            foreach (User user in result)
             {
-                System.Console.Write($"{ u.FirstName} {u.LastName}\n");
+               Console.Write($"{ user.FirstName} {user.LastName}\n");
```
Comment (ohthatbaguk) [inline Names/Users.cs:29]: same as line 22

File: Names/Users.cs [hunk 5]
```diff
@@ -1,40 +1,34 @@
...
-            var result = restoredUser.OrderBy(u => u.FirstName).ThenBy(u => u.LastName);
-
-            System.Console.WriteLine("---------------------------------------");
-            System.Console.WriteLine("After sort:");
-
-            foreach (User u in result)
+            var result = restoredUser.OrderBy(user => user.FirstName).ThenBy(user => user.LastName);
+            Console.WriteLine("--------------------------");
+            Console.WriteLine("After sort:");
+            foreach (User user in result)
```
Comment (ohthatbaguk) [inline Names/Users.cs:27]: use var

File: Names/Models/User.cs (hunk 1 above)
Comment (SimaLoktseva) [reply Names/Models/User.cs:6]: If I understand correctly, is it so:
image

Comment (ohthatbaguk) [reply Names/Models/User.cs:6]: no )
  public string FirstName { get; set; }
  public string LastName { get; set; }  

======>

  public string FirstName { get; set; }

  public string LastName { get; set; }

File: Names Test/Users.cs [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            foreach (var user in restoredUser)
+            {
+                Console.WriteLine($"{ user.GetFullName()}");
+            }
+            var sortedUsers = restoredUser.OrderBy(user => user.FirstName).ThenBy(user => user.LastName);
+            Console.WriteLine("--------------------------");
+            Console.WriteLine("After sort:");
+            foreach (var user in sortedUsers)
+            {
+                Console.WriteLine($"{ user.GetFullName()}");
```
Comment (ohthatbaguk) [inline Names Test/Users.cs:29]: GetFullName returns string so we don’t need interpolation here

**Pull request merged**